climpred/comparisons.py

```diff
@@ -193,7 +210,7 @@ def _e2r(ds, reference):
     return forecast, reference
 
 
-def _m2r(ds, reference):
+def _m2r(ds, reference, stack=True):
     """
     Compares each member individually to a reference in HindcastEnsemble.
 
```

Comment:
Add description of stack here for developers. Can mention that it isn't used here but is placed for consistency in calling the functions.